Composer/packages/client/src/components/CreationFlow/v2/TemplateDetailView.tsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable react/no-danger */
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React from 'react';
+
+type TemplateDetailViewProps = {
+  templateId: string;
+  readMe: string;
+};
+
+export const TemplateDetailView: React.FC<TemplateDetailViewProps> = (props) => {
+  return <div className="content" dangerouslySetInnerHTML={{ __html: props.readMe }}></div>;
```

Comment:
We really should not `dangerouslySetInnerHTML`

Reply:
Yeah Im not a big fan of it for the xss attack vulnerability although we control the src it pulls the html from. 

We would have to rethink how we grab the template detail view. I like the idea of grabbing the readme as it makes template development easier and it sticks with the npm standard. 

Alternatively, the feed sends metadata that could pass the core info of the template (i.e. supported languages, description, etc) to Composer for rendering. It would make visual styling easier and more consistent then pulling readMe markdown.

@benbrown what do you think the meta data config would be in package.json of the template. So instead of populating a readme they would populate a handful of fields in the package.json of their template package.

Reply:
Resolving as the new readMe approach has been added that takes call logic out of the component layer, adds readme to app state and does not dangerouslyinjecthtml